### ISSUE TITLE
Support schema.org embedded metadata

### ISSUE THREAD

**Issue by dietervu** [clarin-eric/VirtualCollectionRegistry#65]
Requires: addition to landing pages, creation of Sitemap, mention Sitemap in robots.txt

**Comment by WillemElbers** [comment]
If DOIs are minted via DOI apis, this comes automatically. See: https://github.com/clarin-eric/VirtualCollectionRegistry/issues/65

When minting PIDs without DOI, this would be nice to have.

Synchronise with VLO developments.

**Comment by WillemElbers** [comment]
Implemented as of https://github.com/clarin-eric/VirtualCollectionRegistry/releases/tag/1.5.0-rc3
